File: core/src/com/jediterm/core/input/Event.java
package com.jediterm.core.input;

final class Event {
  public static final int SHIFT_MASK          = 1;
  public static final int ALT_MASK            = 1 << 3;
  public static final int CTRL_MASK           = 1 << 1;
  public static final int META_MASK           = 1 << 2;
}

File: jediterm-core-mpp/src/jvmMain/kotlin/com/jediterm/core/input/Event.kt
package com.jediterm.core.input

internal object Event {
    const val SHIFT_MASK: Int = 1
    @JvmField
    val ALT_MASK: Int = 1 shl 3
    @JvmField
    val CTRL_MASK: Int = 1 shl 1
    @JvmField
    val META_MASK: Int = 1 shl 2
}